test/ui/ui_unit.js

```diff
@@ -70,6 +70,59 @@ describe('UI', () => {
       });
     });
 
+    describe('set up with one container and src=', () => {
+      /** @type {!HTMLElement} */
+      let container;
+      /** @type {!HTMLVideoElement} */
+      let video;
+
+      beforeEach(async () => {
+        container =
+          /** @type {!HTMLElement} */ (document.createElement('div'));
+        document.body.appendChild(container);
```

Comment:
Shouldn't we remove the container from DOM after test?

Reply:
```
  afterEach(async () => {
    shaka.media.ManifestParser.unregisterParserByMime(fakeMimeType);
    await UiUtils.cleanupUI();
  });
```

UiUtils.cleanupUI() will scan the DOM and remove everything with data tags indicating they belong to Shaka.